Assets/Scripts/DefaultFileBeadsProvider.cs

```diff
@@ -1,49 +1,50 @@
+using System;
 using System.Collections.Generic;
 using UnityEngine;
 
-public class DefaultFileBeadsProvider: ILinkBeadsProvider
+public class DefaultFileBeadsProvider : ILinkBeadsProvider
 {
-        private readonly TextAsset _linkPointsTextFile;
+    private readonly TextAsset _linkPointsTextFile;
 
-        public DefaultFileBeadsProvider(int crossingNumber, int ordering, int numComponents = 1)
-        {
-                _linkPointsTextFile = DefaultFileAccessor.GetTextAsset(crossingNumber, ordering, numComponents);                
-        }
+    public DefaultFileBeadsProvider(int crossingNumber, int ordering, int numComponents = 1)
+    {
+        _linkPointsTextFile = DefaultFileAccessor.GetTextAsset(crossingNumber, ordering, numComponents);
+    }
+
+    public List<Vector3[]> GetBeadsList()
+    {
+        var componentPoints = new List<Vector3>();
+        var linkList = new List<Vector3[]>();
 
-        public List<Vector3[]> GetBeadsList()
+        var linkTextFileText = _linkPointsTextFile.text;
+        var points = linkTextFileText.Split(Environment.NewLine.ToCharArray());
+
+        foreach (var line in points)
         {
-                var componentPoints = new List<Vector3>();
-                var linkList = new List<Vector3[]>();
-
-                var linkTextFileText = _linkPointsTextFile.text;
-                var points = linkTextFileText.Split(System.Environment.NewLine.ToCharArray());
-                
-                foreach(var line in points)
-                {
-                        var point = ParseTextFileLineIntoVector(line);
-
-                        if (point != null) componentPoints.Add(point.Value);
-
-                        if (string.IsNullOrWhiteSpace(line))
-                        {
-                                linkList.Add(componentPoints.ToArray());
-                                componentPoints = new List<Vector3>();
-                        }
-                }
-
-                return linkList;
+            var point = ParseTextFileLineIntoVector(line);
+
+            if (point != null) componentPoints.Add(point.Value);
+
+            if (string.IsNullOrWhiteSpace(line))
+            {
+                linkList.Add(componentPoints.ToArray());
+                componentPoints = new List<Vector3>();
+            }
         }
 
-        private static Vector3? ParseTextFileLineIntoVector(string line)
-        {
-                var coordinates = line.Split(' ');
+        return linkList;
+    }
 
-                if (coordinates.Length != 3) return null;
+    private static Vector3? ParseTextFileLineIntoVector(string line)
+    {
+        var coordinates = line.Split(' ');
```

Comment:
Since this is very dependent on the formatting of the text file, we should include instructions on proper formatting

Reply:
Sure. I'll do that.

Reply:
Actually I checked and those instructions are already included in the Resources/PointsFiles/Default folder.